dashboard/app/models/levels/blockly.rb

```diff
@@ -593,29 +593,39 @@ def localized_function_blocks(blocks)
     return block_xml.serialize(save_with: XML_OPTIONS).strip
   end
 
-  # Localize placeholder texts in text blocks
-  def localized_text_blocks(blocks)
+  # Localizing placeholder texts in certain block types
+  def localized_blocks_with_placeholder_texts(blocks)
```

Comment:
I'm wondering if we should add documentation of what a `placeholder_text` is and maybe a link to a an example block which uses it? We could also just put raw XML in the comments.

I know not many methods have any documentation at all, but it might be a good idea to save the knowledge you gained investigating this task.

I leave it up to you to decide if you think this is worth documenting and also where the right place for the documentation is.